lintcheck/src/json.rs

```diff
@@ -282,9 +273,9 @@ mod test {
             changed: vec![],
         }];
 
-        let expected = "| Lint                                       | Added   | Removed | Changed |
-| ------------------------------------------ | ------: | ------: | ------: |
-| [`clippy::float_arithmetic`](#user-content-float-arithmetic)   |       0 |       0 |       0 |";
+        let expected = "| Lint                                                         | Added | Removed | Changed |
+|--------------------------------------------------------------|-------|---------|---------|
```

Comment:
Don't added back that fancy `---:`

Reply:
Right alignment is pretty nice for number columns, the markdown is rendered on the summary page e.g. here at https://github.com/rust-lang/rust-clippy/actions/runs/12297760099?pr=13820

I believe the print in the run itself is more for debugging, personally I don't think it's worth adding a dependency for

Reply:
Well, added dep, yes, but lintcheck didn't distributed to users and even cached in CI.